roles/StackStorm.nginx/vars/redhat.yml

```diff
@@ -1,3 +1,4 @@
 selinux_dependencies:
   - libsemanage-python
   - libselinux-python
+  - libselinux-python3
```

Comment:
That looks now even more weird now.
If we install `libselinux-python`, why do we need `libselinux-python3` then?

Feels like we're doing something wrong.

Reply:
And to recap, selinux lib is used just for this:
https://github.com/StackStorm/ansible-st2/blob/d58746051d7b1aa7b574afdf1cecf506e7df34d9/roles/StackStorm.nginx/tasks/nginx_redhat.yml#L88-L95

Reply:
According to https://docs.ansible.com/ansible/2.5/modules/seboolean_module.html#requirements

>The below requirements are needed on the host that executes this module.
>
>    libselinux-python
>    libsemanage-python

It looks like python3 module is not needed at all in this situation.

Reply:
I've done a bit more searching... I don't believe nginx needs libselinux-python3.

I've taken it off the system, and everything is still functioning. I

However...

If we have in our code or any code that gets imported an "import selinux" then it's going to fail on the move to python3 without the include of libselinux-python3. But I've searched and none of the packs in StackStorm exchange do this explicitly (and we don't include it in the st2 repo either). 

ansible itself will still use python2, and it's a well reported one of failing if libselinux-python is not installed. So I believe we will want to keep the libselinux-python. It's also a dependency of policycoreutils-python, which is needed for the semanage call. So libselinux-python would be required.

I'm going to take libselinux-python3 out for now. And I'll do the same in the bash installer so that it's same environment.

Reply:
Thanks for the research, @amanda11!

This should also shed more light about the pack dependencies topic: 
https://github.com/StackStorm/st2-packages/issues/241
https://github.com/StackStorm/st2docs/issues/979#issuecomment-621767347